org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/translator/A4Solution.java

```diff
@@ -102,6 +115,9 @@
  * This class stores a SATISFIABLE or UNSATISFIABLE solution. It is also used as
  * a staging area for the solver before generating the solution. Once solve()
  * has been called, then this object becomes immutable after that.
+ *
+ * @modified: Nuno Macedo, Eduardo Pessoa // [HASLab] electrum-temporal,
```

Comment:
adapted to support temporal problems; runtime errors include:
- setting an unbounded `steps` scope with a bounded solver